src/main/java/frisbee/puf/backend/network/ServerThread.java

```diff
@@ -117,7 +124,26 @@ private void startGame(SocketRequest clientRequest) {
         this.otherClient.sendToClient(clientRequest);
     }
 
+    @SneakyThrows
+    private void disconnect(SocketRequest clientRequest) {
+        log.info(client.getInetAddress().getLocalHost() + "has sent a message of type DISCONNECT.");
+        // remove this client thread from client map
+        SocketServer.removeClient(this.teamName, this);
+        // if other client initialized the disconnect, this.otherclient was simultaneously set to null and doesn't need
+        // to be informed again about the disconnect
+        if (this.otherClient == null) {
```

Comment:
Alternative: 
`if (this.otherClient != null) { ... der ganze Rest }`

Reply:
ist ja egal, oder nicht?

Reply:
Du kannst dir damit ein else sparen - IntelliJ hat es mir angemeckert, aber im Prinzip ja.

Reply:
stimmt. ja, ist schöner